src/app/dashboard/dashboard.component.html

```diff
@@ -34,15 +38,19 @@
           </mat-card-content>
         </mat-card>
       </div>
-      <div flex-gt-sm="40">
-        <mat-card>
+      <div flex-gt-xs="40">
+        <mat-card @appear>
           <mat-card-title>Recent News</mat-card-title>
           <mat-divider></mat-divider>
           <app-articles
             [maxArticles]="maxArticles">
           </app-articles>
```

Comment:
There's a horizontal line missing here that should separate the list from the card actions (https://teradata.github.io/covalent/#/design-patterns)